libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -374,6 +374,7 @@ singleton AP_Mission method num_commands uint16_t
 singleton AP_Mission method get_item boolean uint16_t'skip_check mavlink_mission_item_int_t'Null
 singleton AP_Mission method set_item boolean uint16_t'skip_check mavlink_mission_item_int_t
 singleton AP_Mission method clear boolean
+singleton AP_Mission method cmd_has_location boolean uint16_t 0 64000
```

Comment:
should just use "uint16_t'skip_check" to not check the range, we want all values to be allowed, there is nothing wrong with a mission command of 64001 and it could be added in the future

Reply:
done